Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -313,6 +313,7 @@ namespace gdjs {
 
     updatePreRender(): void {
       this.updatePositionIfNeeded();
+      this.applyAntialiasing();
```

Comment:
This recreates and reapplies a new `FXAAFilter` every frame, that is not a good idea. Instead, you should call the `applyAntialiasing` function only on construction to apply the initial filter and from the `RuntimeObject` whenever the functions to change antialiasing settings are called.